### ISSUE TITLE
niqe/ilniqe returns values outside of range

### ISSUE THREAD

**Issue by hdrmadness** [chaofengc/IQA-PyTorch#83]
Hello

I am getting large niqe and ilniqe scores, so I put some print statements 

def forward(self, X: torch.Tensor) -> torch.Tensor:
        r"""Computation of NIQE metric.
        Args:
            X: An input tensor. Shape :math:`(N, C, H, W)`.
        Returns:
            Value of niqe metric in [0, 1] range.
        """
        print(X.size())
        score = calculate_niqe(X, self.crop_border, self.test_y_channel, self.pretrained_model_path, self.color_space)
        print("niqe score should be 0 to 1, and we got {}".format(score))
        return score

and I get:

torch.Size([1, 3, 512, 344])                                                             | 1/10 [00:01<00:15,  1.78s/it]
niqe score should be 0 to 1, and we got 5.133647061808286

**Comment by chaofengc** [comment]
Sorry that the document is not correct. The output value of niqe/ilniqe is a positive real value, without explicit upper bound limitation.

**Comment by hdrmadness** [comment]
Thanks for looking into this. Is there a reference in your documentation where the bounds of each metric can be found? I am finding it difficult to correlate scores of different metrics to each other. I attached a list that I am looking at.

[NR-IQA bounds.txt](https://github.com/chaofengc/IQA-PyTorch/files/11626793/NR-IQA.bounds.txt)

**Comment by chaofengc** [comment]
Thanks for your information. I have added it to the [ModelCard](https://github.com/chaofengc/IQA-PyTorch/blob/main/docs/ModelCard.md)

There are several points about the metric bound:
- The absolute value of each metric, including its bound, is usually meaningless. Only the relative scores of different images are meaningful. The absolute value is related to the training data used as well as some math operator such as `sigmoid`. 
- There are no standard settings in IQA currently. Therefore, we do not compare absolute values between different metrics.
- To correlate scores of different metrics to each other, we usually used the PLCC and SRCC which measures their linear correlations or ranking correlations.
